crates/sui-types/src/sui_system_state/epoch_start_sui_system_state.rs

```diff
@@ -91,7 +91,7 @@ impl EpochStartSystemStateV1 {
     pub fn new_for_testing_with_epoch(epoch: EpochId) -> Self {
         Self {
             epoch,
-            protocol_version: ProtocolVersion::MIN.as_u64(),
+            protocol_version: ProtocolVersion::MAX.as_u64(),
```

Comment:
not sure why testing is starting always with min version, that can obviously create issues, I had to change it to avoid a set of errors. What is the expectation here?

Reply:
I agree that max is the better default for tests. @lxfind do you have any comments?

For something this important it would be nice to have a test for every supported version (since they all have to work correctly when the validator is deployed) but it may not be feasible to write all those tests.